packages/lambda-api-tracker/src/__test__/xyz.request.test.ts

```diff
@@ -77,34 +77,4 @@ o.spec('xyz-request', () => {
             'x-linz-request-id': [{ key: 'x-linz-request-id', value: String(res.header(HttpHeader.RequestId)) }],
         });
     });
-
-    o('should not cache WMTSCapabilities', async () => {
```

Comment:
Adding cache control on the inwards request makes no sense, as this is not served to the client.
```
Client -> CloudFront -> XyzLambda -> TileLambda -> CloudFront -> Client
```

Reply:
Are we setting it on the outwards response?